index.html

```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        
+        <!-- ________ *Nick* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Nick</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">Your handle</a>
```

Comment:
Kindly change the twiiter link with your twitter username. You can choose github too, for that, kindly replace `twitter` with `github` in line 122 and 123.
```suggestion
            <a href="https://www.twitter.com/USERNAME" target="_blank">@UserName</a>
```